src/goto-checker/multi_path_symex_only_checker.h

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************\
+
+Module: Goto Checker using Multi-Path Symbolic Execution only
+
+Author: Daniel Kroening, Peter Schrammel
+
+\*******************************************************************/
+
+/// \file
+/// Goto Checker using Multi-Path Symbolic Execution only (no SAT solving)
+
+#ifndef CPROVER_GOTO_CHECKER_MULTI_PATH_SYMEX_ONLY_CHECKER_H
+#define CPROVER_GOTO_CHECKER_MULTI_PATH_SYMEX_ONLY_CHECKER_H
+
+#include "incremental_goto_checker.h"
+
+#include "symex_bmc.h"
+
+class multi_path_symex_only_checkert : public incremental_goto_checkert
+{
+public:
+  multi_path_symex_only_checkert(
+    const optionst &options,
+    ui_message_handlert &ui_message_handler,
+    abstract_goto_modelt &goto_model);
+
+  resultt operator()(propertiest &) override;
```

Comment:
Documentation is required. An undocumented class with an unnamed method isn't usable.